cql3/statements/alter_keyspace_statement.cc

```diff
@@ -130,6 +130,63 @@ bool cql3::statements::alter_keyspace_statement::changes_tablets(query_processor
     return ks.get_replication_strategy().uses_tablets() && !_attrs->get_replication_options().empty();
 }
 
```

Comment:
> A bug has been discovered while trying to ALTER tablets KS and
specifying only 1 out of 2 DCs - the not specified DC's RF has been
zeroed. This is because ALTER tablets KS updated the KS only with the
RF-per-DC mapping specified in the ALTER tablets KS statement, so if a
DC was ommitted, it was assigned a value of RF=0.

This will be unreadable to a newcomer due to acronym overload.

Reply:
It's KS + DC + RF. I assumed these are well known (in the db env). 
Newcomers have hard time anyways...

Reply:
FWIW: I added all to https://www.notion.so/Dictionary-of-Acronyms-and-Abbreviations-7574566194c14393b2f948855981eef3

Reply:
I'd argue that KS, DC, RF should be in https://opensource.docs.scylladb.com/stable/reference/glossary.html

Reply:
https://github.com/scylladb/scylladb/issues/20415

Reply:
It's not reasonable to have to read something with a glossary in hand.

Acronyms tend to become meaningless symbols.

Reply:
So what's the decision, should I just expand the acronyms? I was just tired myself of writing "replication factor" over and over again, but still wanted to provide a detalied explanation.
Look how much characters I saved, I'm using this number of abbreviations: 4 x KS + 4 x DC + 3 x RF = 22 characters as opposed to 4 x keyspace + 4 x datacenter + 3 x replication factor = 32 + 40 + 54 = 126 characters, that's 1:6 compression - wow! ;)

Reply:
If it's in the same comment I think you can use e.g. `datacenter (DC) ... replication factor (RF) ` the first time and later on use the acronym that you've introduced earlier in line. FWIW, keyspace can stays as is IMO (or everything.  saving characters is secondary to confusing the reader)

Reply:
Sounds reasonable, thanks.

Reply:
Applied the suggestion.